modules/layers/src/text-layer/text-layer.js

```diff
@@ -180,17 +179,20 @@ export default class TextLayer extends CompositeLayer {
   // Text strings are variable width objects
   // Returns the index at the start of each string (every character is rendered by one instance)
   _updateText() {
-    const {data} = this.props;
+    const {data, characterSet} = this.props;
     const textBuffer = data.attributes && data.attributes.getText;
     let {getText} = this.props;
     let {startIndices} = data;
     let numInstances;
 
+    const autoCharacterSet = characterSet === 'auto' && new Set(DEFAULT_CHAR_SET);
```

Comment:
If we are going to automatically generate a character set, do we need to include `DEFAULT_CHAR_SET`?